deploy/aws/deploy/bin/deploy-secrets.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+read -p "What secret name? " secret_name
+
+read -p "What stage? (staging, production) " stage
+
+echo "Set up necessary environment variables"
+read -p "Write MONGO_URI "  mongo_uri
+
+read -p "Write MONGO_DB_NAME " mongo_db_name
+
+read -p "Write API_URL " api_url
+
+read -p "Write WEB_URL " web_url
```

Comment:
let's remove it, because we need to use this script for updating env variables in existing application